ui/lib/core/addon/components/form-field.hbs

```diff
@@ -187,17 +187,38 @@
       @value={{or (get @model this.valuePath) @attr.options.defaultValue}}
       @onChange={{this.onChangeWithEvent}}
     />
+  {{else if (eq @attr.options.editType "toggleButton")}}
+    {{! Togglable Input }}
+    <Toggle
+      @name="toggle-{{@attr.name}}"
+      @onChange={{this.toggleButton}}
+      @checked={{this.toggleInput}}
+      data-test-toggle={{@attr.name}}
+    >
+      <span class="has-text-weight-bold is-large">{{this.labelString}}</span><br />
+      <div class="description has-text-grey">
+        {{#if this.toggleInput}}
+          <span>
+            {{@attr.options.subText}}
+          </span>
+        {{else}}
+          <span>
+            {{or @attr.options.defaultSubText "Vault will use the engine default."}}
```

Comment:
Since FormFields is used in _a lot_ of places, I think this default message should probably even be more general. 
```suggestion
            {{or @attr.options.defaultSubText "Vault will use the system default."}}
```